### DISCUSSION TITLE
Status of stitches?

### DISCUSSION THREAD

**Discussion by csantos-nydig** [stitchesjs/stitches#927] [Help]
Hey guys, I just wanted to check what are you currently focused on.

This repository has been extremely quiet lately. no replies back in Issues nor in Pull requests from the maintainers;

So, it'd be nice to know where are currently putting your effort towards?

We'd appreciate any updates ;) and thanks for this nice library.

**Comment by lukemorales** [comment]
I just asked @peduarte on Twitter, here's the answer:
![CleanShot 2022-01-20 at 11 10 20](https://user-images.githubusercontent.com/14251143/150354735-1407b354-64be-49a7-8f0b-896cc2c5e2c6.png)

**Reply by csantos-nydig** [reply]
Well it seems that @peduarte is not longer working @ Modulz :/
